### PR TITLE
webhook clarification

### PR DESCRIPTION
**Pull request by strouja**
i spent 3 hours trying to get my webook step to work without having port 443 traffic forwarded to the IP address of my Raspberry Pi4 running Home Assistant and was not successful.  I only got this to work after I created a port forwarding rule on my fios router to send all TCP traffic on port 443 to my Raspberry Pi4.   

I do not know of we also need to do make this change (I did make this change too) I added port 8123 to the external url of my dyndns entry in `/config/configuration.yaml`
```
homeassistant:
  external_url: "https://<my-home-dns>.duckdns.org:8123"
```
But I made both of these changes to get my Smarthings Integration to complete and get past the webhook step.    I suspect i only needed to do the port 443 forwarding rule and not make external url use port  8123 as I think it was already doing that before I added 8123 to the external_url

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @strouja, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#19666).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/smartthings.markdown [hunk 1]
```diff
@@ -63,7 +63,7 @@ The PAT is used to create a Home Assistant SmartApp in your SmartThings account
 This integration requires an internet accessible incoming webhook to receive push updates from SmartThings. The preferred approach is to subscribe to [Home Assistant Cloud (Nabu Casa)](https://www.nabucasa.com/) and the integration will configure and use a cloudhook automatically. Alternatively, you will have to configure and setup an internet accessible webhook in Home Assistant as described below:
 
 1. Setup [remote access](/docs/configuration/remote/) via a domain name secured with SSL. *Self-signed SSL certificates are not supported by the SmartThings Cloud API.*
-1. Set the external URL in the Home Assistant [configuration](/docs/configuration/basic) to the URL that Home Assistant is available on the internet (this must start with `https://`).
+1. Set the external URL in the Home Assistant [configuration](/docs/configuration/basic) to the URL that Home Assistant is available on the internet (this must start with `https://`). This must be port 443.  So if you do not use Nabu Casa you must configure your network to allow TCP traffic from the internet on port 443 to reach the IP address of the device running Home Assistant.
```
Comment (frenck) [inline source/_integrations/smartthings.markdown]:
```suggestion
1. Set the external URL in the Home Assistant [configuration](/docs/configuration/basic) to the URL that Home Assistant is available on the internet (this must start with `https://`). This must be port 443. If you do not use Nabu Casa you must configure your network to allow TCP traffic from the internet on port 443 to reach the IP address of the device running Home Assistant.
```

**Review by frenck — APPROVED** [review]
Thanks, @strouja 👍 

Happy Hacktoberfest 🎉 

../Frenck

**Comment by strouja** [comment]
Yes happy Octoberfest to you as well.  Thanks for approving and publishing and I agree with your addition about Nobu Casa.
I went to the public pages and see the change there, so it all looks good, take care

**Pull request merged**